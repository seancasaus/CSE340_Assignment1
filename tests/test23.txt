fR8WO (W)|((H).(y)) , P7iP4 I , dSEuh (0)|((M).((2)*)) , C2eeb c , VDCc8 ((k).(O)).(T) #
" kOTkOTHyHy cW0kOTHykOTIM " 
