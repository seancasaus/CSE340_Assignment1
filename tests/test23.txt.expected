VDCc8 , "kOT"
VDCc8 , "kOT"
fR8WO , "Hy"
fR8WO , "Hy"
C2eeb , "c"
fR8WO , "W"
dSEuh , "0"
VDCc8 , "kOT"
fR8WO , "Hy"
VDCc8 , "kOT"
P7iP4 , "I"
dSEuh , "M"
